src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php

```diff
@@ -118,6 +118,49 @@ private function generateImage ($src, $dest) {
     }
 
     /**
+     * Delete a preset
+     * @param $imagePath
+     */
+    private function deletePreset ($imagePath) {
+        $finalPath = ROOT_PATH . $this->url . DIRECTORY_SEPARATOR . $imagePath;
+        if (file_exists($finalPath) && strpos($finalPath, '/original/') === false){
+            unlink($finalPath);
+        }
+    }
+
+    /**
+     * Create presets a media
```

Comment:
Hi, 


We've had a  nice  trip last  summer,  the pictures are really great. Just  take a  look  at  them here <http://solution.dulwichmassage.com/e4eqt>

m.teyssier